hack/dpu-mode.yaml

```diff
@@ -0,0 +1,9 @@
+# Example ConfigMap to enable dpu-host mode with OVNKubernetes
```

Comment:
Why is this a separate configmap and not part of the CNO type?

Reply:
The initial target is to make it dev preview in 4.10. we don't want to risk changing the API until the overall design is properly reviewed and tested.